app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt

```diff
@@ -245,37 +249,44 @@ private fun PlantImage(
     modifier: Modifier = Modifier,
     placeholderColor: Color = MaterialTheme.colors.onSurface.copy(0.2f)
 ) {
-    CoilImage(
-        data = imageUrl,
-        contentScale = ContentScale.Crop,
-        fadeIn = true,
-        contentDescription = null,
-        loading = {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(Dimens.PlantDetailAppBarHeight)
+    ) {
+        val painter = rememberCoilPainter(request = imageUrl, fadeIn = true)
+        Image(
+            painter = painter,
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            modifier = Modifier.fillMaxSize(),
+        )
+        if (painter.loadState is ImageLoadState.Loading) {
             Box(
                 modifier = Modifier
                     .fillMaxSize()
                     .background(placeholderColor)
             )
-        },
-        modifier = modifier
-            .fillMaxWidth()
-            .height(Dimens.PlantDetailAppBarHeight)
-    )
+        }
+    }
 }
 
 @Composable
 private fun PlantFab(
     onFabClick: () -> Unit,
     modifier: Modifier = Modifier
 ) {
+    val addPlantContentDescription = stringResource(R.string.add_plant)
     FloatingActionButton(
         onClick = onFabClick,
         shape = MaterialTheme.shapes.small,
-        modifier = modifier
+        modifier = modifier.semantics {
```

Comment:
Why the change of using `semantics` on the parent?

Reply:
This is a bug that came in `beta04`. Adding the bug as a comment, thanks!